app/screens/settings/notification_settings_mobile/notification_settings_mobile_base.js

```diff
@@ -100,24 +100,24 @@ export default class NotificationSettingsMobileBase extends PureComponent {
             channel,
             comments,
             desktop,
-            desktop_duration,
+            desktop_duration: desktopDuration,
```

Comment:
Why is this being changed?

Reply:
eslint rules update

Reply:
Nice of it to just change the defaults on us since I don't see `.eslintrc.json` changing